hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/connector/infoschema/SqlInfoSchemaTest.java

```diff
@@ -126,8 +126,8 @@ public void test_datalinks() {
         assertRowsAnyOrder(
                 "SELECT * FROM information_schema.datalinks",
                 asList(
-                        new Row("hazelcast", "public", "s_dl", type, "{}", "SQL"),
-                        new Row("hazelcast", "public", "c_dl", type, "{}", "CONFIG")
+                        new Row("hazelcast", "public", "s_dl", type, true, "{}", "SQL"),
+                        new Row("hazelcast", "public", "c_dl", type, true, "{}", "CONFIG")
```

Comment:
@Fly-Style do we return full class name for datalink in information schema? it would be better to return short type

Reply:
https://github.com/hazelcast/hazelcast/pull/24110

Reply:
Fixed in #24111